src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3059,6 +3075,12 @@ void addProperties(ImmutableArray<ParameterSymbol> recordParameters)
                         builder.InstanceInitializersForRecordDeclarationWithParameters.Insert(addedCount, new FieldOrPropertyInitializer.Builder(property.BackingField, paramList.Parameters[param.Ordinal]));
                         addedCount++;
                     }
+                    else if (existingMember.Kind switch { SymbolKind.Property => true, SymbolKind.Field => true, _ => false })
```

Comment:
>Field => true [](start = 98, length = 13)

It is required by [https://github.com/dotnet/csharplang/blob/master/proposals/records.md#properties](https://github.com/dotnet/csharplang/blob/master/proposals/records.md#properties) that there be a property corresponding to each parameter. If you find a field, it should be an error.